torchvision/prototype/transforms/_augment.py

```diff
@@ -82,59 +83,45 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
         else:
             i, j, h, w, v = 0, 0, img_h, img_w, image
 
-        return dict(zip("ijhwv", (i, j, h, w, v)))
+        return dict(i=i, j=j, h=h, w=w, v=v)
 
-    def _transform(self, input: Any, params: Dict[str, Any]) -> Any:
-        if isinstance(input, features.Image):
-            output = F.erase_image_tensor(input, **params)
-            return features.Image.new_like(input, output)
-        elif is_simple_tensor(input):
-            return F.erase_image_tensor(input, **params)
+    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+        if isinstance(inpt, features._Feature):
+            return inpt.erase(**params)
+        elif isinstance(inpt, PIL.Image.Image):
+            # Shouldn't we implement a fallback to tensor ?
```

Comment:
Yes let's create a quick and dirty implementation that casts to tensor and back to PIL similar to what we do at `gaussian_blur`.

Reply:
Change the comment to a TODO.